src/main/java/trello/ui/pages/BoardPage.java

```diff
@@ -75,6 +97,58 @@ public void archiveListByTitle(final String listTitle) {
         archiveListButton.click();
     }
 
+    /**
+     * Adds a new card to a list.
+     * @param listTitle defines the lists where add a card.
+     * @param cardTitle defines the card  to be add.
+     */
+    public void addCardInList(final String listTitle, final String cardTitle) {
+        for (WebElement list : lists) {
+            if (list.findElement(By.className("js-list-name-input")).getText().equals(listTitle)) {
+                if (list.findElement(By.className("js-add-a-card")).isDisplayed()) {
+                    list.findElement(By.className("js-add-a-card")).click();
+                } else {
+                    list.findElement(By.className("js-add-another-card")).click();
+                }
+                titleCard.sendKeys(cardTitle);
```

Comment:
Please the method generic that we have to add a text to element.
E.G: WebDriverMethod.setTextElement(titleCard, cardTitle)

Reply:
Solved